app/models/history.rb

```diff
@@ -108,6 +109,37 @@ class History < ApplicationRecord
       .where(history_type: [HISTORY_TYPES[:reports_reviewed], HISTORY_TYPES[:report_reviewed]])
   }
 
+  def self.errored_report_reminder_group_of_patients(patients: nil, created_by: 'Sara Alert System', comment: 'Failed Contact Attempt', error_message: nil)
+    histories = []
+    patients.uniq.each do |pat|
```

Comment:
Is the `uniq` necessary?

Reply:
No, it is not. Will remove

Reply:
The uniq actually needed because if a responder of SMS Texted Weblink is the subject of the `dispatch_errored_contact_history_items function`, the patients sent to  errored_report_reminder_group_of_patients will be `[patient, patient&.responder]`.